external/mlsql-ets/src/main/java/tech/mlsql/plugins/ets/SyntaxAnalyzeExt.scala

```diff
@@ -0,0 +1,40 @@
+package tech.mlsql.plugins.ets
+
+import tech.mlsql.common.utils.log.Logging
+import org.apache.spark.ml.param.Param
+import org.apache.spark.ml.util.Identifiable
+import org.apache.spark.sql.execution.MLSQLAuthParser
+import org.apache.spark.sql.expressions.UserDefinedFunction
+import org.apache.spark.sql.{DataFrame, SparkSession}
+import streaming.dsl.ScriptSQLExec
+import streaming.dsl.mmlib.SQLAlg
+import streaming.dsl.mmlib.algs.param.WowParams
+import streaming.log.WowLog
+
+/**
+  * 09/08/2021 hellozepp(lisheng.zhanglin@163.com)
+  */
+class SyntaxAnalyzeExt(override val uid: String) extends SQLAlg with WowParams with Logging with WowLog {
+  def this() = this(Identifiable.randomUID("tech.mlsql.plugins.ets.SyntaxAnalyzeExt"))
+
+  override def train(df: DataFrame, path: String, params: Map[String, String]): DataFrame = {
+    val context = ScriptSQLExec.contextGetOrForTest()
+    val _sql = params(sql.name)
+    val identifiers = MLSQLAuthParser.filterTables(_sql, context.execListener.sparkSession)
+    logInfo(format(identifiers.map(_.table).toList.mkString("[",",","]")))
```

Comment:
So we just log the tables we extract from sql?  How can we get the result ?

Reply:
Need to be converted to DF, I will fix it